tests/a100_generate_container_image_test.py

```diff
@@ -47,6 +48,90 @@ def get_image_id(tag):
     return ''
 
 
+def remove_images(repo_name):
+    """
+    Remove any existing repo images.
+    WARNING: This is destructive.
+    """
+
+    # Remove {repo_name}:* images.
+    images = subprocess.check_output(['docker', 'images', '-q', repo_name], stderr=None, encoding='utf-8').strip().split('\n')
+    for myimg in images:
+        #   pylint: disable=subprocess-run-check
+        subprocess.run(['docker', 'image', 'rm', myimg, '--force'], stderr=subprocess.DEVNULL, stdout=subprocess.DEVNULL, check=False)
```

Comment:
I think this is very dangerous for the user - it will remove all `dunes` images, even the running ones.
Remember, we are attaching those tests to the release as well.
I preferred the previous solution, where `bootstrap.py` had to be executed manually (or by CI script). And even if it was executed, it did not alter running images. What was the issue with it?

Reply:
A number of the existing tests are also destructive and will destroy a container. I believe the latest commit (with `destructive` marking) corrects this issue.